CHANGELOG.md

```diff
@@ -1,237 +1,368 @@
+<!-- markdownlint-disable MD024 -->
```

Comment:
I am a bit stuck here with ignoring the rules.
Can we somehow disable the rule MD024 since now our headers represent categories like `### Added` and they will be duplicated?